WordPress/Classes/Models/Suggestion.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+
+@objcMembers public class Suggestion: NSObject {
```

Comment:
While we're here, could we just add a a comment explaining what this is used for?